task/export-image-tar/0.1/export-image-tar.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: export-image-tar
+spec:
+  params:
+    - name: input-image
+      type: string
+      description: The input image that is to be saved as an oci-archive tar file
+    - name: tar-filename
+      default: "image.tar"
+      description: The tar filename of the saved input image
+      type: string
+    - name: dest-image-tag
+      default: ""
+      description: The tag of the destination image when the tar file is loaded. Use this if you want to hide the source repository name, or if you have issues with digest missmatch which can happen sometimes with `podman save` and `podman load`
+      type: string
+    - name: output-artifact
+      type: string
+      description: The output url that points to the OCI artifact of the tar image
+  results:
+    - description: Digest of the tar artifact just pushed
+      name: ARTIFACT_DIGEST
+    - description: Image repository where the tar artifact was pushed
+      name: ARTIFACT_URL
+  steps:
+    - name: save-tar
+      image: registry.access.redhat.com/ubi9/podman:latest
+      script: |
+        podman --version
+
+        podman pull $(params.input-image)
```

Comment:
How should this work when the image is multi-arch? With this command, it will only pull the _one_ image matching the system's architecture (i.e. x86). In the future, if we are going to change over to arm node pools by default, this would effectively be a breaking change as it would pull a different image (if that is even possible).

Reply:
@arewm Oh that is a really good point. I could add the an `arch` parameter to pass through to `podman pull --arch ARCH`

Reply:
@arewm Ok I added an `input-arch` param to specify a specific arch to pull if it's different from the host arch.